components/textfield/stories/textarea.template.js

```diff
@@ -4,14 +4,13 @@ import { Template as Textfield } from "./template";
 
 export const Template = ({
 	customClasses = [],
-	rootClass = "spectrum-Textfield",
+	rootClass = "spectrum-Textfield--multiline",
 	size = "m",
 	multiline = true,
 	...item
 } = {}, context = {}) => Textfield({
 	customClasses: [
 		rootClass,
-		typeof size !== "undefined" ? `${rootClass}--size${size.toUpperCase()}` : null,
```

Comment:
This seemed to be adding an additional `.spectrum-Textfield--multiline--sizeM` once I started editing the text area `rootClass`. We don't have sizing for `.spectrum-Textfield--multiline`/text area, so does this make sense?